From 89b24c5fd23e449434a959fb01b54bcf5c5194fd Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Tue, 1 Jul 2014 18:38:44 -0400
Subject: [redhat] kernel.spec: move some builddeps around a bit

Message-id: <20140701183844.GE26859@redhat.com>
Patchwork-id: 85160
O-Subject: [RHEL7 PATCH] redhat/kernel.spec: move some builddeps around a bit
Bugzilla: 1000180 1056320
RH-Acked-by: Neil Horman <nhorman@redhat.com>

Bugzilla 1000180 and bugzilla 1056320.

python-devel is needed for more than just perf, looks like maybe the same
is true of new-devel and perl(ExtUtils::Embed). This'll solve some
--with{,out} combinations not building, per the two referenced bugzillas,
and really doesn't harm us any, so why not...

---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 5f83b10..c9831c6 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -290,6 +290,7 @@ BuildRequires: hostname, net-tools, bc
 BuildRequires: xmlto, asciidoc
 BuildRequires: openssl
 BuildRequires: hmaccalc
+BuildRequires: python-devel, newt-devel, perl(ExtUtils::Embed)
 %ifarch x86_64
 BuildRequires: pesign >= 0.109-4
 %endif
@@ -297,7 +298,7 @@ BuildRequires: pesign >= 0.109-4
 BuildRequires: sparse >= 0.4.1
 %endif
 %if %{with_perf}
-BuildRequires: elfutils-devel zlib-devel binutils-devel newt-devel python-devel perl(ExtUtils::Embed) bison
+BuildRequires: elfutils-devel zlib-devel binutils-devel bison
 BuildRequires: audit-libs-devel
 %ifnarch s390 s390x
 BuildRequires: numactl-devel
-- 
1.7.1